examples/general-use/README.md

```diff
@@ -0,0 +1,36 @@
+### Using the Terraform Action from `oaknational/oak-terraform-actions`
+
+To use the Terraform action from the `oaknational/oak-terraform-actions` repository in your GitHub workflows, follow these steps:
+
+1. **Create a Workflow File**: In your target repository, create a `.github/workflows` directory if it doesn't exist.
+
+2. **Add Workflow Configuration**: Create a workflow file (e.g., `main.yml`) with the following content:
+
+```yaml
+name: oak-terraform-action
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  example-job:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
```

Comment:
Do we need to checkout here? There is a checkout step as the first step of the action itself.

Oh and they are diff versions checkout@v2 (here) or checkout@v3 (in the yaml)

Reply:
I'd keep these actions separate, and assume people still want to checkout themselves. This gives a bit more flexibility. I've made this more consistent, and will update the README too.

Reply:
Had another think about this - and might be one for later. One to come back to for my own sake:

1. We rarely have a sole project for infra/terraform. Instead, there's a directory in a project, and we likely want to deploy Terraform code first and then the code. Think Cloud Run functions. 
2. Therefore, we want to have a checkout. One for the future is to put this action to _only run_ when a folder has an `/infrastructure` directory.